Project Structure/Project Structure.md

```diff
@@ -36,11 +36,11 @@ For classes that are shared across multiple screens, a top-level `Shared` folder
 
 In general, leave the files that Xcode generates with new project creation where they are. This can be helpful for the following reasons:
 
-* Keeps the `AppDelegate` at the top-level
+* Keeps the `AppDelegate`, `SceneDelegate`, and `App` at the top-level
```

Comment:
Not sure if it's worth delineating UIKit and SwiftUI like we have elsewhere?

Reply:
I know AppDelegate and SceneDelegate can still be used in a SwiftUI app so delineating SwiftUI for "App" should be good enough here.